custom_components/nest_protect/manifest.json

```diff
@@ -1,18 +1,16 @@
 {
-    "domain": "nest_protect",
```

Comment:
Do we need the indentation changes in this file?

Reply:
This is the default indentation of vsstudio and vscode for json files. Reverted to your indentation